src/Raven.Server/Documents/Handlers/Admin/RachisAdminHandler.cs

```diff
@@ -335,6 +336,16 @@ public async Task AddNode()
             {
                 requestExecutor.DefaultTimeout = ServerStore.Engine.OperationTimeout;
 
+                // test connection to remote.
+                var result = await ServerStore.TestConnectionToRemote(nodeUrl, database: null);
+                if (result.Success)
```

Comment:
where is the error handling on the `Success == false`?

Reply:
fixed